src/agent/CommandLockAll.ts

```diff
@@ -19,7 +18,6 @@ class CommandLockAll extends CommandPolykey {
       const { default: PolykeyClient } = await import(
         'polykey/dist/PolykeyClient'
       );
-      const { WebSocketClient } = await import('@matrixai/ws');
       const clientUtils = await import('polykey/dist/client/utils');
```

Comment:
Are `clientUtils` even used anywhere? it should be removed if not.

Reply:
Removing them.

Reply:
This was all listed in `npm run lintfix`, but it does not automatically remove them. @addievo please keep track of any warnings that come up when you do it.